concordia/views.py

```diff
@@ -1563,7 +1568,7 @@ def filter_and_order_reviewable_assets(
 @never_cache
 @atomic
 def redirect_to_next_reviewable_asset(request):
-    campaign = Campaign.objects.published().order_by("ordering")[0]
+    campaign = Campaign.objects.published().listed().order_by("ordering")[0]
```

Comment:
This reminds me that we might want to handle the case where `[0]` would fail if the queryset was empty but I'm not sure whether it's worth changing it to use`.first()` and checking the result since we haven't had this problem yet.